cmd/deployment.go

```diff
@@ -112,7 +112,7 @@ func newDeploymentRootCmd(client *houston.Client, out io.Writer) *cobra.Command
 		newDeploymentListCmd(client, out),
 		newDeploymentUpdateCmd(client, out),
 		newDeploymentDeleteCmd(client, out),
-		newLogsCmd(),
+		newLogsCmd(client),
```

Comment:
This is more of a general comment, but why don't we use a singleton variable inside the `houston` package to set a "global" HTTP Client `client`, instead of having to pass it everywhere in the project as a method parameter ?

Like:
```go
package "houston"

var Client *http.Client

// and initialize this variable either by setting it directly in the cmd when the program is initialized ?
```

I was also thinking about a different way to do it, but once again this is a more general discussion, outside the scope of the PR:
Right now, we're using the `houston` package as a low level HTTP client for the Houston API, each cmd has to re-create its own houston.Request object etc....

Why don't we make it a higher level abstraction over the Houston API ? It could expose methods like `FetchLogs(component string) (*returnType[], error)` or `CreateUser(email, password string) (*UserResponse, error)`, and all of the underlying implementations (like the HTTP Client and each cmd having to format the Request itself) could be done inside the `houston` package.

I've described this comment in the following notion document: https://www.notion.so/astronomerio/Review-CLI-27067e25a2314686b42d37e0bd9bf73e

What you do think @andriisoldatenko ?

Reply:
Some good things about this approach:
- each component is de-coupled, the CLI needs something from the Houston API, it uses the houston package, gives it few parameters, and receive the response object without any knowledge about the underlying implementation or HTTP CLient manipulations.
- We can easily create a `type interface` for the houston package, and mock it for unit tests, instead of directly testing the http Responses in these CMD tests.
- We can write unit tests for the houston package independently and test the http behaviours.

Reply:
as discussed this change could be a part of separate PR